tests/integration/components/o-s-s/currency-input-test.ts

```diff
@@ -16,6 +16,44 @@ module('Integration | Component | o-s-s/currency-input', function (hooks) {
     assert.dom('.currency-input-container').exists();
   });
 
+  test('it renders currency only', async function (assert) {
+    this.onChange = () => {};
+    await render(hbs`<OSS::CurrencyInput @currency="USD" @onlyCurrency={{true}} @onChange={{this.onChange}} />`);
+
+    assert.dom('.currency-input-container').exists();
+    assert.dom('.currency-selector').exists();
+
+    const currencyLabel = await findAll('.currency-selector span');
+
+    assert.dom(currencyLabel[0]).exists();
+    assert.dom(currencyLabel[0]).hasText('$');
+
+    assert.dom(currencyLabel[1]).exists();
+    assert.dom(currencyLabel[1]).hasText('USD');
+
+    assert.dom('.currency-input input').doesNotExist();
+  });
+
+  test('it renders currency only with empty currency param', async function (assert) {
+    this.onChange = () => {};
+    await render(
+      hbs`<OSS::CurrencyInput @currency={{undefined}} @onlyCurrency={{true}} @onChange={{this.onChange}} />`
+    );
+
+    assert.dom('.currency-input-container').exists();
+    assert.dom('.currency-selector').exists();
+
+    const currencyLabel = await findAll('.currency-selector span');
+
+    assert.dom(currencyLabel[0]).exists();
+    assert.dom(currencyLabel[0]).hasText('$');
+
+    assert.dom(currencyLabel[1]).exists();
+    assert.dom(currencyLabel[1]).hasText('USD');
```

Comment:
maybe a `assert.dom('.currency-selector').hasText('$ USD')` can replace all this?  🤷‍♂️ (double check but it's good anyway)